pkg/asset/machines/master.go

```diff
@@ -514,7 +512,7 @@ func (m *Master) Machines() ([]machineapi.Machine, error) {
 	libvirtapi.AddToScheme(scheme)
 	openstackapi.AddToScheme(scheme)
 	ovirtproviderapi.AddToScheme(scheme)
-	vsphereapi.AddToScheme(scheme)
+	machineapi.AddToScheme(scheme)
```

Comment:
This is not an adequate replacement. The scheme from github.com/openshift/api/machine/v1beta1 does not include the provider specs [1]. If we want to use the types defined in openshift/api, then we will need to manual add the vSphere provider spec to the scheme.

[1] https://github.com/openshift/api/blob/master/machine/v1beta1/register.go#L34-L41

Reply:
I agree. I was aware of the CI failure and was looking for a solution for this.

Reply:
Added the vspherespec into scheme.